kratos/python/add_vector_to_python.cpp

```diff
@@ -35,7 +35,7 @@ namespace Python
 
     template< typename TVectorType > py::class_< TVectorType > CreateVectorInterface(pybind11::module& m, std::string Name )
     {
-        py::class_< TVectorType, std::shared_ptr<TVectorType> > binder(m,Name.c_str());
+        py::class_< TVectorType, std::shared_ptr<TVectorType> > binder(m,Name.c_str(), py::buffer_protocol());
```

Comment:
hm this is basically what I meant in the other PR with the `EquationIdVector` (where passing by reference did not work yet)
I think this is a larger change than we think

Reply:
do you see it as problematic?

Reply:
personally i would see as more problematic the possibility of automatically converting between vectors and matices (we have the same for matrices)

Reply:
the point is about vector assignement

i can do without this by the use of a Set function, howwever this one is needed to write things on the line of 

x[:] = other_data

which is something often needed in solvers